lib/beaker-hostgenerator/data.rb

```diff
@@ -685,6 +685,22 @@ def osinfo
             'platform' => 'fedora-14-i386',
           },
         },
+        'freebsd12-64' => {
+          :general => {
+            'platform' => 'freebsd-12-amd64',
+          },
+          :vagrant => {
+            'box' => 'generic/freebsd12',
```

Comment:
In https://github.com/voxpupuli/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/hypervisor/vagrant.rb we already do similar conversions automatically so you don't have to maintain a manual mapping and you get it "for free" when FreeBSD 14 comes out..

Reply:
you're saying I can omit the `vagrant => {}` section?